functions/src/flags.ts

```diff
@@ -12,3 +12,8 @@ export namespace Flags {
   export const isEmulator = process.env.FUNCTIONS_EMULATOR === 'true'
   export const requireInvitationOrganizationToMatchSsoProviderId = false
 }
+
+export namespace Constants {
+  export const webFrontendBaseUrl =
+    process.env.WEB_FRONTEND_BASE_URL ?? 'https://engagehf-mock-base-url.com'
```

Comment:
What is the best way that we can eventually inject this in the production setup. I suspect something we can set on the Firebase side? Otherwise, would it make sense to make this part of the static data or seeding mechanism?

Reply:
My assumption would be to inject it in the CI pipeline, does that make sense? Then, we could simply inject one of three different urls into the system depending on the workflow `environment`, right?

Reply:
Would that persist when we deploy it?

Reply:
Mhmm, okay - valid point - we will need some way to persist this for the Twilio setup as well though in #238, so we can technically just inject a source file containing all these constants, right?

Reply:
I changed it to use https://firebase.google.com/docs/functions/config-env?gen=2nd to inject the environment values. It still builds and runs fine even without me injecting values, which is somewhat weird to me, since the documentation makes it seem that this wouldn't be the case... but let's see

Reply:
Thank you!